setup.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(
+    name='condor-comforter',
+    version='0.1',
+    description="Some helper scripts for running HTCondor jobs",
+    author='Robin Aggleton',
+    author_email='',
+    url='https://github.com/BristolComputing/condor-comforter',
+    py_modules=['cmsRunCondor.cmsRunCondor', 'haddaway.haddaway'],
+    scripts=['cmsRunCondor/cmsRunCondor.py', 'haddaway/haddaway.py'],
+    install_requires=['htcondenser>=0.3.0'],
+    dependency_links=['git+https://github.com/raggleton/htcondenser#egg=htcondenser-0.3.0']
```

Comment:
requirement is `>=0.3.0` but you are always installing it into `egg=htcondenser-0.3.0`?
Why not `egg=htcondenser`?